### PR TITLE
Dropped PHP 8.0 support

### PR DESCRIPTION
**Pull request by kukulich**

### CODE REVIEW

File: infection.bootstrap.php [hunk 1]
```diff
@@ -2,10 +2,4 @@
...
 declare(strict_types=1);
 
-require __DIR__ . '/stubs/UnitEnum.php';
-require __DIR__ . '/stubs/BackedEnum.php';
-
-require __DIR__ . '/stubs/ReflectionEnum.php';
-require __DIR__ . '/stubs/ReflectionEnumUnitCase.php';
-require __DIR__ . '/stubs/ReflectionEnumBackedCase.php';
-require __DIR__ . '/stubs/ReflectionIntersectionType.php';
+// Nothing currently
```
Comment (Ocramius) [inline infection.bootstrap.php]: If unused, kill it 😁

File: psalm.bootstrap.php [hunk 2]
```diff
@@ -2,10 +2,4 @@
...
 declare(strict_types=1);
 
-require __DIR__ . '/stubs/UnitEnum.php';
-require __DIR__ . '/stubs/BackedEnum.php';
-
-require __DIR__ . '/stubs/ReflectionEnum.php';
-require __DIR__ . '/stubs/ReflectionEnumUnitCase.php';
-require __DIR__ . '/stubs/ReflectionEnumBackedCase.php';
-require __DIR__ . '/stubs/ReflectionIntersectionType.php';
+// Nothing currently
```
Comment (Ocramius) [inline psalm.bootstrap.php]: If unused let's kill it

File: src/Reflection/Adapter/ReflectionEnumBackedCase.php [hunk 3]
```diff
@@ -14,6 +14,7 @@
 use function array_map;
 use function sprintf;
 
+/** @psalm-suppress MissingImmutableAnnotation */
```
Comment (Ocramius) [inline src/Reflection/Adapter/ReflectionEnumBackedCase.php:17]: Can we attempt hunting for immutability? 🤔

Comment (kukulich) [reply src/Reflection/Adapter/ReflectionEnumBackedCase.php:17]: It's probably complicated, we already have https://github.com/Roave/BetterReflection/blob/6.5.x/src/Reflection/Adapter/ReflectionParameter.php#L26 and https://github.com/Roave/BetterReflection/blob/6.5.x/src/Reflection/Adapter/ReflectionNamedType.php#L12

File: infection.bootstrap.php (hunk 1 above)
Comment (kukulich) [reply infection.bootstrap.php]: Done.

File: psalm.bootstrap.php (hunk 2 above)
Comment (kukulich) [reply psalm.bootstrap.php]: Done.

File: src/Reflection/Adapter/ReflectionEnumBackedCase.php (hunk 3 above)
Comment (Ocramius) [reply src/Reflection/Adapter/ReflectionEnumBackedCase.php:17]: OK, let's roll with this as-is then, thanks :)

**Review by Ocramius — APPROVED** [review]
Thanks @kukulich! 

Dropping a release with this, mostly to set a clean baseline for anyone upgrading.

File: composer.json [hunk 4]
```diff
@@ -3,7 +3,7 @@
     "description": "Better Reflection - an improved code reflection API",
     "license": "MIT",
     "require": {
-        "php": "~8.0.12 || ~8.1.0 || ~8.2.0",
```
Comment (Ocramius) [inline composer.json:6]: Ah, just realized that we didn't update `composer.lock` :| 

Running Renovate with that now.

**Pull request merged**